### DISCUSSION TITLE
Formula rendering issues

### DISCUSSION THREAD

**Discussion by L2Fish** [PDConSec/vsc-print#258] [Q&A] [answered]
This is my first attempt to submit an issue here.

I wrote a formula in markdown and printed it with this plugin.

`$$\hat a={\frac{m\omega}{2\hbar}}$$`

There seems to be something wrong with the formula rendering here.

![image](https://github.com/PDConSec/vsc-print/assets/93245002/e384a647-f5e5-4127-9fc7-5711068e02d5)

Each formula will have two results in HTML.

**Comment by PeterWone** [comment]
There are several Markdown extensions that support LaTeX like this. I tested with https://marketplace.visualstudio.com/items?itemName=mathpix.vscode-mathpix-markdown using the following markdown and got the results below.

```md
# test.md

$$\hat a={\frac{m\omega}{2\hbar}}$$
```
![image](https://github.com/PDConSec/vsc-print/assets/5498936/0403da10-86f3-4c0a-9180-2479121f57db)

The nature of the problem is almost certainly down to incompatibility in the stylesheets injected by whatever extension you're using and the stylesheets I use. Due to a number of issues like this I plan to stop using the built-in rendering pipeline and instead load a selection of superior Markdown extensions. For example, the one I linked above also handles SMILES chemistry notation.

**Comment by L2Fish** [comment, marked as answer]
In my case, the extension responsible for rendering the LaTeX is **Markdown+Math**. The double rendering was resolved by adding a stylesheet reference to settings, as per the documentation which specifically describes the use of this extension.

```json
"markdown.styles": [
	"https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css"
]
```
Now this plugin is working!
